freebasics/migrations/0001_initial.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '__latest__'),
+        ('polls', '__latest__'),
```

Comment:
I think you should pin this to actual migrations.. because in future you might not want it to do this